edx_sga/sga.py

```diff
@@ -174,7 +176,11 @@ def student_view(self, context=None):
         """
         context = {
             "student_state": json.dumps(self.student_state()),
-            "id": self.location.name.replace('.', '_')
+            "id": self.location.name.replace('.', '_'),
+            "max_file_size": getattr(
+                settings, "STUDENT_FILEUPLOAD_MAX_SIZE",
```

Comment:
Have you declared this configuration variable (STUDENT_FILEUPLOAD_MAX_SIZE) somewhere in settings?